wscript

```diff
@@ -77,14 +81,18 @@ def configure(ctx):
     ctx.load('compiler_cxx')
 
     # Check required headers
-    req_headers = ['stdlib.h', 'string.h', 'unistd.h', 'stdint.h', 'stdio.h', 'jpeglib.h']
+    req_headers = ['stdlib.h', 'string.h', 'stdint.h', 'stdio.h']
+    if platform.system() != "Windows":
+        req_headers += ['dlsym.h', 'unistd.h', 'jpeglib.h']
+    else:
+        req_headers += ['windows.h']
     for header in req_headers:
         ctx.check_cc(header_name = header, auto_add_header_name = True, mandatory = True)
 
     # Check for required libs
-    req_libs = [('m', 'm'), ('jpeg', 'jpeg')]
-    for (lib, uselib) in req_libs:
-        ctx.check_cc(lib = lib, uselib_store = uselib)
+    # req_libs = [('m', 'm'), ('jpeg', 'jpeg')]
+    # for (lib, uselib) in req_libs:
+    #     ctx.check_cc(lib = lib, uselib_store = uselib)
```

Comment:
"This is not the ~~droid~~ code you are looking for?" :)

Reply:
Yep, thanks for pointing this out.